src/components/dashboard/reporting/checklist/ModernChecklistForm.jsx

```
@@ -400,21 +414,21 @@ const ModernChecklistForm = ({
-        const dateValue = value ? 
-          (value instanceof Date ? value.toISOString().split('T')[0] : 
-           typeof value === 'string' && value.includes('T') ? value.split('T')[0] : 
+        const dateValue = value ?
+          (value instanceof Date ? value.toISOString().split('T')[0] :
+           typeof value === 'string' && value.includes('T') ? value.split('T')[0] :
            value) : '';
-        
+
         return (
           <input
             type="date"
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)